hack/dump/logging-dump.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+set -x
```

Comment:
use this instead
```
set -euo pipefail
if [ -n "${DEBUG:-}" ] ; then
    set -x
fi
```
Note that with `-u` this script may throw a lot of errors, so you may have to add a lot of `"${var:-}" to check for unassigned variables.